### DISCUSSION TITLE
How to configure Master/Slave mode via CLI

### DISCUSSION THREAD

**Discussion by Chandra-Nagarajan** [TPC-Council/HammerDB#167] [Q&A] [answered]
Is it possible to configure Master/Slave mode via CLI? If yes, Can you please guide me how it can be done? I searched for how to set up in CLI but couldn't find any document on that.

Thanks,
Chandra

**Comment by sm-shaw** [comment, marked as answer]
This has already been added as per Issue #18. It is included in the next v4.0 release which once approved by the TPC will be made available.  Note that as per issue #152 the terminology  has been changed from Master/Slave to Primary/Replica however the functionality remains the same.

**Comment by sm-shaw** [comment]
Closed as functionality is in next release.
